src/cf-forms/usage.md

```diff
@@ -110,16 +86,34 @@ See the 'Form icons' section below for guidance on adding icons to states.
        title="Test input">
 ```
 
-### Input icons
-
-Form input icons add small positioning tweaks to Capital Framework icons.
+### Inline Form Validation
 
-<input type="text" value="" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-email"></span>
+<div class="m-field m-field__error">
+    <label class="a-label__heading" for="form-input-error">Label</label>
+    <input class="a-text-input a-text-input__error"
+           type="text"
+           value="Invalid input"
+           id="form-input-error"
+           aria-describedby="form-input-error_message">
+    <div class="a-error-message" id="form-input-error_message" role="alert">
```

Comment:
If the only message is an error message, I think `a-error-message` is a little more compact. (This is also how the USWDS do it.) Are there other inline messages we need to consider that should be included in the whole framework and not a page-by-page rule?